src/NuProj.Targets/NuProj.targets

```diff
@@ -1009,7 +1009,16 @@
 
     =================================================================================================================== -->
 
-  <Target Name="Clean" DependsOnTargets="AssignProjectConfiguration;EstablishNuGetPaths">
+  <PropertyGroup>
+    <CleanDependsOn>
+      BeforeClean;
+      AssignProjectConfiguration;
+      EstablishNuGetPaths
```

Comment:
Fixed.
I don't see ending semicolons in any place in this file, and I don't think it is relevant to this PR.

For instance:
```
DependsOnTargets="AssignNuProjPackageDependenciesTargetFramework;GetNuGetPackageDependencies"
```